pyrato/__init__.py

```diff
@@ -1,48 +1,26 @@
 # -*- coding: utf-8 -*-
 
 __author__ = \
-    """Marco Berzborn - Institute for Hearing Technology and Acoustics"""
-__email__ = 'marco.berzborn@akustik.rwth-aachen.de'
+    """The pyfar developers"""
+__email__ = 'info@pyfar.org'
 __version__ = '0.3.2'
 
-from .rap import (
-    reverberation_time_linear_regression
-)
-from .roomacoustics import (
-    reverberation_time_energy_decay_curve,
-    energy_decay_curve_analytic,
+
+from .parametric import (
     air_attenuation_coefficient,
 )
```

Comment:
```suggestion
```

how about the air attenuation? Maybe we can remove it since it will be part of the next pyfar release https://github.com/pyfar/pyfar/pull/734, and this is a pr for v1.0.0, isnt it?

maybe we can change ``energy_decay_curve_analytic`` in a way that it just uses air_attenuation_factor as input (remove frequency, air_absorption). then the calculation is upto the user